tests/keras_tests/feature_networks_tests/test_features_runner.py

```diff
@@ -159,6 +159,7 @@
 from tests.keras_tests.feature_networks_tests.feature_networks.conv_func_substitutions_test import \
     ConvFuncSubstitutionsTest
 from model_compression_toolkit.qat.common.qat_config import TrainingMethod
+from model_compression_toolkit.target_platform_capabilities.constants import BIAS, KERAS_KERNEL
```

Comment:
I think that the changes in this file are unnecessary. If so, please remove them